wiki/osu!_wiki/Contribution_guide/Best_practices/de.md

```diff
@@ -0,0 +1,101 @@
+# Bewährte Methoden
+
+Diese Seite beschäftigt sich mit einigen Aufgaben, die beim Beitragen zu erledigen sind. Die hier verfolgten Ansätze sind dazu gedacht, den Prozess einfacher zu machen und können auch auf andere Projekte angewandt werden, die auf GitHub oder einer ähnlichen Plattform untergebracht sind.
+
+## Einführung
+
+*Für mehr Informationen über git und GitHub, siehe [GitHub-Dokumentation (auf Englisch)](https://docs.github.com)*
+
+**Git** ist ein System zur Versionsverwaltung, das dabei hilft, Änderungen an Dateien zu verwalten. Die Daten und Änderungsgeschichte des osu!-Wikis werden in einem Git-Repository gespeichert. **GitHub** ist eine Entwicklungsplattform, die eine Webschnittstelle für Git-Repositorys bietet und eine Reihe von Projektmanagement-Tools bereitstellt.
+
+## Die Verzweigung synchronisieren
+
+Um Änderungen an einem Repository auf GitHub vorzunehmen, muss sich ein Mitwirkender eine isolierte Kopie des Repositorys beschaffen, welche *Verzweigung* (Fork) genannt wird. Wenn du deine Verzweigung des Repositorys `osu-wiki` erstellst, fertigst du eine Momentaufnahme des Inhalts zu diesem Augenblick an. **Synchronisiere immer deine Verzweigung** zum Beitragen, bevor du Änderungen vornimmst — das kann direkt in GitHub gemacht werden:
+
+1. Gehe zu deiner Verzweigung des Repositorys `osu-wiki`.
+2. Wähle den Branch `master` aus dem Dropdown-Menü aus.
+3. Klicke auf `Fetch upstream` und wähle `Fetch and merge`.
+
+![](img/update-branch.png "Den nicht mehr aktuellen Branch aktualisieren")
+
+Jetzt ist dein Branch auf dem neusten Stand mit dem ursprünglichen Repository.
```

Comment:
```suggestion
Jetzt ist dein Branch auf dem selben Stand wie das ursprüngliche Repository.
```

🤔

Reply:
Sounds good, though `dem selben` should be one word I think.

Reply:
Fixed.